[32mINFO    [0m pytest_operator.plugin:plugin.py:675 Connecting to existing model microk8s-localhost:test on unspecified cloud
[33mWARNING [0m juju.client.connection:connection.py:927 unexpected facade UserSecretsDrain received from the controller
[33mWARNING [0m juju.client.connection:connection.py:927 unexpected facade UserSecretsManager received from the controller
[32mINFO    [0m integration.test_charm:test_charm.py:48 Deploying mysql, mysqlrouter and application
[32mINFO    [0m juju.model:model.py:2069 Deploying local:jammy/mysql-router-k8s-0
[32mINFO    [0m juju.model:model.py:2069 Deploying ch:amd64/jammy/mysql-test-app-63
[32mINFO    [0m juju.model:model.py:2069 Deploying ch:amd64/jammy/mysql-k8s-210
[32mINFO    [0m integration.test_charm:test_charm.py:79 Waiting for mysqlrouter to be in BlockedStatus
[32mINFO    [0m pytest_operator.plugin:plugin.py:790 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.4.3    unsupported  02:49:33Z

App               Version                  Status   Scale  Charm             Channel      Rev  Address         Exposed  Message
mysql-k8s         8.0.39-0ubuntu0.22.04.1  active       3  mysql-k8s         8.0/edge     210  10.152.183.81   no       
mysql-router-k8s                           waiting      1  mysql-router-k8s                 0  10.152.183.138  no       installing agent
mysql-test-app    0.0.2                    waiting      1  mysql-test-app    latest/edge   63  10.152.183.251  no       installing agent

Unit                 Workload  Agent      Address      Ports  Message
mysql-k8s/0          active    idle       10.1.124.77         
mysql-k8s/1*         active    executing  10.1.124.79         Primary
mysql-k8s/2          active    idle       10.1.124.78         
mysql-router-k8s/0*  unknown   idle       10.1.124.72         
mysql-test-app/0*    waiting   idle       10.1.124.73         

[32mINFO    [0m pytest_operator.plugin:plugin.py:796 Juju error logs:

unit-mysql-k8s-1: 02:45:01 ERROR unit.mysql-k8s/1.juju-log Failed to get member state: mysqld daemon is down

[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Forgetting main...